src/sentry/static/sentry/app/views/settings/project/projectOwnership/index.jsx

```diff
@@ -114,6 +114,33 @@ class ProjectOwnership extends AsyncView {
             ]}
           />
         </Form>
+
+        <Form
+          apiEndpoint={`/projects/${organization.slug}/${project.slug}/ownership/`}
```

Comment:
While it's probably ok to have this setting be specific in the ownership category, we generally have put options into ProjectOption in the past. I'll defer to the team here but having many ways to configure a project will create complexity and maintenance concerns down the road.

Reply:
Sure, just let me know which design you prefer and I will change it accordingly. I thought there was some parity with the `fallthrough` setting, but having it as an option in (I guess) `General Settings -> Event Settings` also makes perfect sense.